experimental/packages/exporter-trace-otlp-http/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@opentelemetry/exporter-trace-otlp-http",
   "version": "0.27.0",
+  "private": "true",
```

Comment:
really?

Reply:
This was to prevent it from being published in stable. Now that it's back to experimental it can be removed.